components/CommonHeader.tsx

```diff
@@ -0,0 +1,107 @@
+import Toolbar from '@mui/material/Toolbar'
+import Typography from '@mui/material/Typography'
+import * as React from 'react'
+import { theme } from 'theme/theme'
+
+import OSLogo from '../assets/darkThemeLogo.svg'
+
+const pages = ['About', 'Projects', 'Partner', 'Volunteer', 'Events']
+
+const CommonHeader = () => {
+  // React states and functions for handling the hamburger menu.
+  const [anchorElNav, setAnchorElNav] = React.useState<null | HTMLElement>(null)
+
+  const handleOpenNavMenu = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorElNav(event.currentTarget)
+  }
+
+  const handleCloseNavMenu = () => {
+    setAnchorElNav(null)
+  }
+
+  return (
+    <AppBar position="static" sx={{ background: theme.palette.primary.main }}>
+      <Container maxWidth="xl">
+        <Toolbar sx={{ height: '50px' }}>
+          {/* LOGO */}
+          <img
+            src={OSLogo.src}
+            style={{
+              height: '80%',
+              width: '120px',
+              margin: '0px 60px 0px 0px',
+            }}
+          />
+          {/* Hamburger menu when the screen is small. */}
+          <Box sx={{ flexGrow: 1, display: { xs: 'flex', md: 'none' } }}>
+            <IconButton
+              size="large"
+              aria-label="page-info"
+              aria-controls="menu-appbar"
+              aria-haspopup="true"
+              onClick={handleOpenNavMenu}
+              sx={{ color: theme.palette.primary.contrastText }}
+            >
+              <MenuIcon />
+            </IconButton>
+            <Menu
+              id="menu-appbar"
+              anchorEl={anchorElNav}
+              anchorOrigin={{
+                vertical: 'bottom',
+                horizontal: 'left',
+              }}
+              keepMounted
+              transformOrigin={{
+                vertical: 'top',
+                horizontal: 'left',
+              }}
+              open={Boolean(anchorElNav)}
+              onClose={handleCloseNavMenu}
+              sx={{
+                display: { xs: 'block', md: 'none' },
+              }}
+            >
+              {pages.map((page) => (
+                <MenuItem
+                  key={page}
+                  onClick={handleCloseNavMenu}
+                  style={{ borderRadius: '0px' }}
+                >
+                  <Typography textAlign="center">{page}</Typography>
+                </MenuItem>
+              ))}
+            </Menu>
+          </Box>
+          {/* Menu items that are shown on desktop */}
+          <Box sx={{ flexGrow: 1, display: { xs: 'none', md: 'flex' } }}>
+            {pages.map((page) => (
+              <Button
+                key={page}
+                onClick={handleCloseNavMenu}
+                sx={{
+                  color: theme.palette.primary.contrastText,
+                  borderRadius: '0px',
+                }}
+                disableRipple={true}
+              >
+                {page}
+              </Button>
+            ))}
```

Comment:
Do we want to include the 'Support Us' button here at this point?
We will have to disable the tailwind CSS for it to display properly:
            <Button
              variant="contained"
              color="secondary"
              onClick={handleCloseNavMenu}
            >
              Support Us
            </Button>

Reply:
I purposely left it out because afaik we don't have a way for people to donate to us ... yet?